src/components/chat.tsx

```diff
@@ -451,7 +451,27 @@ function shouldTriggerFeedback(
   const lastMessage = sinceLastUserMessage[sinceLastUserMessage.length - 1];
   const parsed = parseOutput(lastMessage.content);
 
-  return lastMessage.role === "assistant" && parsed.tellUser.length > 0;
+  // Were all function messages errors?
+  const allErrors = sinceLastUserMessage.every(
+    (chat) => chat.role === "function" && functionMessageIsError(chat.content),
+  );
+
```

Comment:
isn't this only true if there are no assistant messages? Aren't there normally assistant message?